scan/src/rpc/TxHook.re

```diff
@@ -20,6 +20,17 @@ module Event = {
 
   let decodeEvents = json =>
     List.flatten(JsonUtils.Decode.(json |> field("events", list(decodeEvent))));
+
+  let getValueOfKey = (events: list(t), key) => {
+    let hasEvent = events->Belt_List.some(event => event.key == key);
```

Comment:
Let's do this. 
`events->Belt.List.keepMap((event) => event.key == key ? Some(event.value) : None)->Belt.List.get(0)`